pkg/planner/cardinality/pseudo.go

```diff
@@ -36,6 +36,10 @@ const (
 // If one condition can't be calculated, we will assume that the selectivity of this condition is 0.8.
 const selectionFactor = 0.8
 
+const (
+	TiDBOptSelectivityFactor = "tidb_opt_selectivity_factor"
```

Comment:
Should we establish a relevant system variable like other factors? For example, ```TiDBOptTableRowIDScanCostFactor```.

Reply:
Oh, I separate a new code block and make a new variable named "selectivity factor". Also I add some comments above of this block.
If this name is OK ?